### DISCUSSION TITLE
Document dependencies?

### DISCUSSION THREAD

**Discussion by cartok** [asdf-community/asdf-php#107] [General]
Hi can't we document dependencies just in a similar way as [asdf does](https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies)?

Example:
| Packagemanager | Command  |
| - | - |
| Pacman | sudo pacman -S autoconf bison curl gettext git openssl pkgconf re2c postgresql-libs php-tidy |

**Comment by paulera** [comment]
This would be great, I have been in a very tedious cycle of `asdf install` -> `"missing whatever"` -> `apt-cache search` -> `apt-get install` -> `rinse & repeat`.
However, different PHP versions have different dependencies. How could we cater for all different ones, including the packages version back when they were released?

**Comment by paulera** [comment]
This page has a fair decent list for PHP 7.4: https://hub.docker.com/r/jabs/docker-php7-fpm/dockerfile

This is a list of packages I had to install using `apt-get`, but there might be some missing from this list if I already had installed them before for another program
`sudo apt-get install mlocate libbz2-dev libssl-dev autoconf bison re2c pkg-config libxml2-dev libcurl4-openssl-dev libgd-dev libonig-dev libpq-dev libzip-dev`
